### DISCUSSION TITLE
김기창의 데이터 모델링 강의/김기창

### DISCUSSION THREAD

**Discussion by jintaeyeong** [jintaeyeong/Project#13] [Book 정보]
프로젝트 데이터 설계를 위한 자료를 찾아보며 이 책에 대한 정보가 나와있어서 첨부합니다.
추후 읽어보면 좋을 것 같습니다. 

이 책을 읽어보신 분이 계신다면 어떻게 읽으셨는지 궁금합니다.

[김기창의 데이터 모델링 강의](https://product.kyobobook.co.kr/detail/S000200383533)

![image](https://github.com/user-attachments/assets/2afd6b3e-49ac-4e6f-afce-4195d5a8acd4)

**Comment by tttghost** [comment]
읽어보지 못했습니다. 😢
